src/App.css

```
@@ -1,107 +1,25 @@
 .App {
-  text-align: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
```

```suggestion
    justify-content: space-between;
```
This will help make sure modal shows.

---

Good catch!